csc-667-fall-2022-sfsu-roberts/term-project-tons
# Implement user registration API

Implements user registration API.

- Added `GET /api/users/:userId` route.
- Added `POST /api/users` route.

**To test:**
1. `npm run db:rollback:all`
2. `npm run db:migrate`
3. `npm run start:dev:win` or `npm run start:dev` if not on Windows
4. Send an HTTP request to the API (examples in screenshots below)
5. Check the database to see if the user was inserted.

### Creating user
![image](https://user-images.githubusercontent.com/7808054/198796275-bb41482f-fb1f-406b-9f7b-52a89aacec27.png)

### Fetching user
![image](https://user-images.githubusercontent.com/7808054/198798654-4a75ee15-d635-48e7-b388-461f7ab543a0.png)
